web-components/src/components/menu/scss/menu-item.scss

```diff
@@ -0,0 +1,40 @@
+.md-menu-item {
+  list-style: none;
+  padding: rem-calc(2);
+
+  a {
+    color: var(--menu-default, $lm-menu-default-light);
+    display: flex;
+    padding: 6px 12px;
```

Comment:
create scss variables for these pixel values in the empty file you created: 
`web-components/src/components/menu/scss/settings.scss`

Reply:
Fixed